shark-android/src/main/java/shark/AndroidObjectInspectors.kt

```diff
@@ -52,12 +53,15 @@ enum class AndroidObjectInspectors : ObjectInspector {
       reporter.whenInstanceOf("android.view.View") { instance ->
         // This skips edge cases like Toast$TN.mNextView holding on to an unattached and unparented
         // next toast view
-        val mParentRef = instance["android.view.View", "mParent"]!!.value
+        val mParentRef =
+          instance.getOrThrow("android.view.View", "mParent").value
```

Comment:
Obfuscation shouldn't impact classes from the Android SDK. This isn't a bad idea but should likely be a different method that throws a different exception (which states that it looks like AOSP might have changed)

Reply:
same comment applies for a lot of the calls here.